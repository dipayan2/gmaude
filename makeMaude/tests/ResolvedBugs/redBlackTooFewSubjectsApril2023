#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Main
export MAUDE_LIB

../../src/Main/maude \
  < $srcdir/redBlackTooFewSubjectsApril2023.maude -no-banner -no-advise \
  > redBlackTooFewSubjectsApril2023.out 2>&1

diff $srcdir/redBlackTooFewSubjectsApril2023.expected redBlackTooFewSubjectsApril2023.out > /dev/null 2>&1
