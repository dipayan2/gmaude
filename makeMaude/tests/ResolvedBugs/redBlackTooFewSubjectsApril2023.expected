==========================================
reduce in FOO : h(f(0, g(7))) .
rewrites: 30
result Set: h(f(0, 0))
==========================================
reduce in FOO : h(f(0, g(7))) .
rewrites: 30
result Set: h(f(0, 0))
Bye.
